starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -52,4 +48,41 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+
+  @Test
+  // Test for Create -> check # of items
+  public void create() {
+    // Creating two new items to "create"
+    Inventory item1 = new Inventory();
+    Inventory item2 = new Inventory();
+    item1.setName("item1");
+    item2.setName("item2");
+    item1.setProductType(PRODUCT_TYPE);
+    item2.setProductType(PRODUCT_TYPE);
+    // Call the function you want to test
+    this.inventoryDAO.create(item1);
+    this.inventoryDAO.create(item2);
+    // Make sure that the objects in Inventory == 2
+    Assert.assertEquals(2, this.mongoTemplate.findAll(Inventory.class).size());
```

Comment:
You should be able to combine these tests into one create test by just adding those extra asserts at the end here.